tests/functional/codegen/modules/test_interface_imports.py

```diff
@@ -79,7 +79,7 @@ def test_multi_import_module_fails(make_input_bundle, get_contract):
 
     input_bundle = make_input_bundle({"lib1.vy": lib1, "lib2.vy": lib2})
     with pytest.raises(StructureException) as e:
-        c = get_contract(main, input_bundle=input_bundle)
+        get_contract(main, input_bundle=input_bundle)
```

Comment:
i think there is a note about this somewhere but we are migrating away from `get_contract` for tests that just check compilation errors -- if we are updating this line, update it to use `compile_code` (or one of its friends)